e2e-tests/playwright/.claude/agents/playwright-test-generator.md

```
@@ -0,0 +1,65 @@
+    - File should contain single test
+    - File name must be fs-friendly scenario name
+    - Test must be placed in a describe matching the top-level test plan item
+    - Test title must match the scenario name
+    - Includes a comment with the step text before each step execution. Do not duplicate comments if step requires
+      multiple actions.
+    - Always use best practices from the log when generating tests.
+
+                 <example-generation>
+                 For following plan:
+
+    ```markdown file=specs/plan.md
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)